res/database/Default/database_scripts.xml

```diff
@@ -374,27 +377,27 @@
 				</titleaction>
 				<descriptionaction>
 					<de>leidet seit dem Kindesalter an %DISEASE%|bestreitet sein Leben mit %DISEASE%|leidet unter %DISEASE%|kann nur singend seine %DISEASE% bekämpfen</de>
-					<en>suffers from %DISEASE% since childhood|makes his living with %DISEASE%|suffers from %DISEASE%|can only fight his %DISEASE% singing</en>
+					<en>has been suffering from %DISEASE% since childhood|lives with %DISEASE%|suffers from %DISEASE%|can only fight his %DISEASE% by singing</en>
```

Comment:
"by singing" is not quite the same as in the German original; "while singing" would be a better fit

Reply:
or 'with singing'?
Only nuances, but fine, I'm indifferent here, ack.